.test-defs/cmd/create-shoot/helper.go

```diff
@@ -0,0 +1,146 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+
+	gardenv1beta1 "github.com/gardener/gardener/pkg/apis/garden/v1beta1"
+	"github.com/gardener/gardener/pkg/client/kubernetes"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// getSeed returns the first seed ordered by creation date with the provided cloudprofile.
+func getSeed(gardenClusterClient kubernetes.Interface, cloudprofileName string) (*gardenv1beta1.Seed, error) {
```

Comment:
Function is not needed, Gardener will automatically determine an appropriate seed cluster itself -> remove